### ISSUE TITLE
New Deploy hangs Node-RED

### ISSUE THREAD

**Issue by CB-IT** [Glogo/node-red-contrib-keypress#1]
Hi,
the keypress node works fine. Thanks for your work.
But if i have to deploy a flow with a running keypress-node node-red cannot stop the flows. Everything hangs up.

Is there any solution?

Best regards

**Comment by parthi2929** [comment]
yes, I am also facing same problem. Its still not solved in 2 years?
